### PR TITLE
add novelty data to sqlite db

### PR DESCRIPTION
**Pull request by chrished**
Add novelty and reuse data based on textual metrics from:

Arts S, Melluso N, Veugelers R (2023). Beyond citations: Text-based metrics for assessing novelty and its impact in scientific publications. https://doi.org/10.48550/arXiv.2309.16437

Paper: https://doi.org/10.48550/arXiv.2309.16437
Data: https://zenodo.org/record/8283353 (DOI: 10.5281/zenodo.8283353)
Code: https://github.com/nicolamelluso/science-novelty

### CODE REVIEW

**Comment by chrished** [comment]
- [ ] adjust loading script to drop existing table
- [ ] add measures from novelty_reuse table to author_output_total and author_output_firstauthor tables
- [ ] adjust author_output table if necessary

**Comment by chrished** [comment]
- [ ] run and test new author_output table

**Comment by chrished** [comment]
@f-hafner anything that should be changed/added?

File: src/dataprep/main/load_novelty_reuse/dump_db.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    new_word_comb_reuse INTEGER,
+    cosine_max REAL,
+    cosine_avg REAL,
+    n_words INTEGER,
+    n_bigrams INTEGER,
+    n_trigrams INTEGER
+);
+.mode csv
+.import "$EXPANDED_CSV_PATH" novelty_reuse
+EOF
```
Comment (f-hafner) [inline src/dataprep/main/load_novelty_reuse/dump_db.sh:36]: can you add an index on `PaperId`?

File: src/dataprep/main/load_novelty_reuse/dump_db.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
```
Comment (f-hafner) [inline src/dataprep/main/load_novelty_reuse/dump_db.sh:1]: I noted in the current db that the first row contains again the column names. I think, if the csv files has headers, these must be dropped when using the `.import`  in sqlite. (I don't remember exactly how to do it, but I think we're doing that in other files too)

File: src/dataprep/temp/prep_linked_data.log [hunk 3]
```diff
@@ -7,16 +7,16 @@ where_stmt_iterations_graduates="\n    WHERE train_name like '%combined%'\n    "
 where_stmt_iterations_advisors="\n    WHERE train_name like '%combined%'\n    "
 current_links for graduates
 current_links for advisors
-Time elapsed: 0.6102594097455343 minutes 
+Time elapsed: 0.30106003681818644 minutes
```
Comment (f-hafner) [inline src/dataprep/temp/prep_linked_data.log:10]: any idea why it is running faster now?

**Review by f-hafner — CHANGES_REQUESTED** [review]
I made some comments.

**Review by f-hafner — APPROVED** [review]

**Comment by f-hafner** [comment]
thanks!

**Pull request merged**